lib/resources/os.rb

```diff
@@ -58,54 +58,47 @@ def to_s
     # Windows release numbers from https://en.wikipedia.org/wiki/Ver_(command)
     # 6.0.6001 6.0.6002
     def windows_server_2008?
-      return true if release =~ /6.0.600/
-      false
+      windows? and release =~ /6.0.600/ ? true : false
```

Comment:
We should use `&&` instead of `and` here. And we can probably eliminate the ternary with something like this:

```ruby
windows? && !release.match(/6.0.600/).nil?
```